### PR TITLE
Enables grabber again, the lower merchants guild members

### PR DESCRIPTION
**Pull request by zakanater19**
## About The Pull Request

Adds four grabbers, lower members of the merchants guild, tested locally

## Why It's Good For The Game

Protection for the merchant

### CODE REVIEW

**Comment by NGGJamie** [comment]
What do these do that the existing two shophands aren't? We just had the PR propose 2 more and said that was too many.

**Comment by zakanater19** [comment]
Well they can't value anything so they cant stand in the shop windows and trade, they can only be muscle and carry things up the balloon or do protection

**Comment by TheManWithNoHands** [comment]
> They can only be muscle and carry things up the balloon or do protection

Those are things anyone can do. There is already the problem of people not interacting much with people outside their job bubble. This just makes it worse. Your making the extra muscels a merchant could buy (have RP interactions with other players) if he even needs it, into a role. 

A big reason why the shophand doesnt ad to this problem, is because the shophand assists the merchant but is also the learning role for that job. They have stuff to do, not any adventurer can do. 

I say, rather add a few spare keys this role would need, to allow the merchant to hire himself muscels more easily.

tl;dr Role bloat bad.

**Comment by OrbitalParadise** [comment]
1 shop hand 1 grabber

**Comment by YosemiteYam** [comment]
FOUR grabbers? That sounds terrible. Just let the merchant hire guards/mercs, they're the only people that got the resources to do so besides the king anyway instead of giving them shitty roles that can only be half-decent guards

**Comment by NGGJamie** [comment]
As mentioned above, shophands basically already fill this role plus help at the windows. If the merchant needs a runner, they can send their shophand, or one of them, to perform the task. Additional roles would be overkill for that purpose.
